unlock-app/src/__tests__/components/interface/Error.test.js

```diff
@@ -1,32 +1,37 @@
 import React from 'react'
 import * as rtl from 'react-testing-library'
-import 'jest-dom/extend-expect'
 
-import { Error } from '../../../components/interface/Error'
+import { Errors, mapStateToProps } from '../../../components/interface/Error'
+import { setError } from '../../../actions/error'
+import errorReducer from '../../../reducers/errorReducer'
 
 const close = jest.fn()
 
 afterEach(rtl.cleanup)
 describe('Error Component', () => {
   describe('when the component has no children or no message', () => {
     it('should not render anything', () => {
-      const wrapper = rtl.render(<Error close={close} />)
+      const wrapper = rtl.render(<Errors close={close} />)
+      expect(wrapper.container.firstChild).toBeNull()
+    })
+    it('should not render when errors length is 0', () => {
+      const wrapper = rtl.render(<Errors close={close} errors={[]} />)
       expect(wrapper.container.firstChild).toBeNull()
     })
   })
 
   describe('when the component has a children', () => {
     it('should display the content of the children', () => {
       const wrapper = rtl.render(
-        <Error close={close}>There was an error.</Error>
+        <Errors close={close}>There was an error.</Errors>
```

Comment:
How about if we remove the legacy ability to display whatever markup is passed as a child to `<Error>`? This will remove the logical inconsistency. I didn't touch these old tests except to rename the component

Reply:
That would work... in another PR!